lib/persistence/relations/articles.rb

```diff
@@ -4,7 +4,7 @@ class Articles < ROM::Relation[:sql]
       schema :articles, infer: true
 
       def published
-        where(published: true)
+        by_published(true)
```

Comment:
Yes, rom creates a generic `by_pk` for you and for every indexed attribute you get `by_*` methods too